### ISSUE TITLE
Typescript files are being parsed incorrectly.

### ISSUE THREAD

**Issue by niemyjski** [sveltejs/eslint-plugin-svelte#1110]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest version of the packages.

### What version of ESLint are you using?

9.21.0

### What version of `eslint-plugin-svelte` are you using?

3.0.2

### What did you do?

This seems to be only happening in typescript svelte files: file.svelte.ts

`Parsing error: Unexpected token {`

```svelte
import type { Login, TokenResult } from './models';
```

### What did you expect to happen?

It doesn't error out and works like 2.x

### What actually happened?

`Parsing error: Unexpected token {`

### Link to **GitHub Repo** with Minimal Reproducible Example

https://github.com/exceptionless/Exceptionless/tree/main/src/Exceptionless.Web/ClientApp

npm install, update dep and run npm run format.

**Comment by niemyjski** [comment]
Do you know what this was a duplicate of or when the fix will be out? Thanks again

**Comment by baseballyama** [comment]
Please read https://github.com/sveltejs/eslint-plugin-svelte/issues/1102
